akka-actor/src/main/scala/akka/pattern/Patterns.scala

```diff
@@ -250,9 +252,262 @@ object Patterns {
     scalaAfter(duration, scheduler)(value.call())(context)
 
   /**
-   * Returns a [[scala.concurrent.Future]] that will be completed with the success or failure of the provided value
+   * Returns a [[scala.concurrent.Future]] that will be completed with the success or failure of the provided Callable
    * after the specified duration.
    */
   def after[T](duration: FiniteDuration, scheduler: Scheduler, context: ExecutionContext, value: Future[T]): Future[T] =
     scalaAfter(duration, scheduler)(value)(context)
 }
+
+object PatternsCS {
+  import akka.japi
+  import akka.actor.{ ActorRef, ActorSystem }
+  import akka.pattern.{ ask ⇒ scalaAsk, pipe ⇒ scalaPipe, gracefulStop ⇒ scalaGracefulStop, after ⇒ scalaAfter }
+  import akka.util.Timeout
+  import scala.concurrent.Future
+  import scala.concurrent.duration._
+
+  /**
+   * <i>Java API for `akka.pattern.ask`:</i>
+   * Sends a message asynchronously and returns a [[scala.concurrent.Future]]
```

Comment:
Yes, my strategy is to make it compile first, then do the textual work while PR validation is running.